woo-razorpay.php

```diff
@@ -1490,6 +1490,7 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
 
             // set default payment method
             $payment_method = $this->id;
+            $payment_method_title = $this->title;
```

Comment:
Didn't we already have some code for setting the title? @ChetanGN

Reply:
But for COD also it was considering the same title i.e `Credit Card/Debit Card/NetBanking `